api

```diff
@@ -427,18 +427,7 @@ Package: $package_name" > ~/$package_name/DEBIAN/control
   
   #install dummy deb
   {
-  #If this app's dummy deb is already insalled, uninstall it
-  if package_installed "$package_name" ;then
-    status "Reinstalling the $package_name package..."
-    apt_lock_wait
-    local output="$(sudo -E apt purge -y $package_name 2>&1 | less_apt | tee /dev/stderr)"
```

Comment:
When developing `install_packages`, I *really* thought that the dummy deb had to be reinstalled for anything to happen. Otherwise, apt would say something like "XXXX is already installed and the latest version"

Reply:
I figured, but that's not the case. Installing over itself works just fine as long as the dummy deb contents have changed

if you try to install it again and the dummy has the same contents as the first time, yes you just get this 
```Note, selecting 
'dummy' instead of '/home/garrett/dummy.deb'

dummy is already the newest version (1.0).
```

but as long as you change the dependencies then it will always install over itself. thats what a second install_packages would do in a script so its perfect
```
Note, selecting 'dummy' instead of '/home/garrett/dummy.deb'
The following packages will be upgraded:
  dummy
1 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
Need to get 0 B/736 B of archives.
After this operation, 0 B of additional disk space will be used.
Get:1 /home/garrett/dummy.deb dummy all 1.0 [736 B]
(Reading database ... 398236 files and directories currently installed.)
Preparing to unpack /home/garrett/dummy.deb ...
Unpacking dummy (1.0) over (1.0) ...
Setting up dummy (1.0) ...
```